data/Nigeria/Senate/ep-popolo-v1.0.json

```diff
@@ -138,54 +256,141 @@
     {
       "contact_details": [
         {
+          "type": "email",
+          "value": "senmushitu@gmail.com"
+        },
+        {
           "type": "phone",
           "value": "08028505890"
         }
       ],
+      "email": "senmushitu@gmail.com",
       "id": "1d5de848-fe51-4511-8856-1a2090a38c00",
+      "identifiers": [
+        {
+          "identifier": "muhammad-shittu",
+          "scheme": "shineyoureye"
+        }
+      ],
       "image": "http://www.nass.gov.ng/images/mps/813.jpg",
       "images": [
         {
           "url": "http://www.nass.gov.ng/images/mps/813.jpg"
+        },
+        {
+          "url": "http://www.shineyoureye.org/media_root/images/Jigawa_North_East_-_Muhammad_Shittu.jpg"
         }
       ],
-      "name": "MUHAMMAD SHITU"
+      "name": "MUHAMMAD SHITU",
+      "other_names": [
+        {
+          "name": "Muhammad Shittu",
+          "note": "alternate"
+        }
+      ]
     },
     {
+      "birth_date": "1962-12-19",
       "contact_details": [
         {
+          "type": "email",
+          "value": "abs@sarakimail.com bukisaraki@aol.com"
```

Comment:
There are about 5 examples of email fields that are doubled up like this. In the scraper you can separate those with a semicolon to have them flow through as separate values here. (Or, I guess, at a pinch you could update the SELECT to replace on space, but it's probably better to do it in the scraper)